lesson_3/readme.md

```diff
@@ -1,2 +1,42 @@
-# Order:
-1. `word_chain.md`
+# Django Girls Code Camp Week 3
+
+Requirements:
+
+- Finish CodeCademy
+- Attempt Exercises
+
+---
+
+| Time  | Content |
+| ------------- | ------------- |
+| 1:10-1:30  | Overview of the day (Rachell/DY)  |
```

Comment:
I thought we decided to not give them away that day if we're not going to show them how to use them that day